crates/subspace-networking/Cargo.toml

```diff
@@ -16,18 +16,23 @@ include = [
 ]
 
 [dependencies]
+anyhow = "1.0.58"
 async-trait = "0.1.53"
 bytes = "1.1.0"
 derive_more = "0.99.17"
+derive-new = "0.5.9"
```

Comment:
Do we really need the whole crate so we can avoid writing `new` implementation? I'd vote against it.

Reply:
I could remove it for now and return it back if I encounter more trivial constructors.

Reply:
I really don't find it very readable to be completely honest, For trivial cases IDE can generate constructors for you automatically already. For less trivial cases you'll have to learn what the macros tries to do, so explicit code would be preferable there as well.